src/LinuxTracing/LibunwindstackUnwinder.h

```diff
@@ -45,9 +45,15 @@ class LibunwindstackUnwinder {
 
   virtual LibunwindstackResult Unwind(pid_t pid, unwindstack::Maps* maps,
                                       const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
-                                      const void* stack_dump, uint64_t stack_dump_size) = 0;
+                                      const void* stack_dump, uint64_t stack_dump_size,
+                                      bool offline_memory = false,
```

Comment:
I would call this "offline_memory_only" to be clearer, when this is false *some* offline memory is still used.